test/fixtures/property-prefix.js

```diff
@@ -0,0 +1,174 @@
+import {getSupport} from 'caniuse-support'
+
+import {prefix} from '../../src/index'
+
+const levelNumber = {
+  none: 0,
+  partial: 1,
+  full: 2,
+}
+
+const cases = [
```

Comment:
I think I can try relying on https://github.com/postcss/postcss-js for this. Let me see if it works.